src/foundation/components/FrozenBanner/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { cx } from '@linaria/core';
+import React from 'react';
+
+import { frozenBannerContainer } from './css';
+import { useLocalProps } from '../../hooks/useLocalProps';
+import { SBUFoundationProps } from '../../types';
+import { Label } from '../Label';
+
+type Props = SBUFoundationProps<{
+  label?: string;
+}>;
+export const FrozenNotification = ({
+  className,
+  label = 'Channel frozen',
+  testId = 'sendbird-frozen-notification',
```

Comment:
why is it called testId?

Reply:
https://sendbird.slack.com/archives/C050Z6DA70A/p1713159557929769